app/scripts/modules/google/src/loadBalancer/configure/http/backendService/backendService.component.js

```diff
@@ -46,6 +46,14 @@ module(GOOGLE_LOADBALANCER_CONFIGURE_HTTP_BACKENDSERVICE_BACKENDSERVICE_COMPONEN
           .value();
       };
 
+      this.getSessionAffinitySuggestions = () => {
+        if (!this.loadBalancer.isInternal) {
```

Comment:
Similar to the comment in `app/scripts/modules/google/src/loadBalancer/configure/http/backendService/backendService.component.html`, can we check the load balancer type explicitly here rather than using the `isInternal` bool?

Reply:
(Same comment to everywhere reading `isInternal`.)